cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,21 +1,41 @@
+import _ = require("cypress/types/lodash");
+import {MenuContentPage} from "../page/index";
+import {ProductsListPage} from "../page/index";
+import {ShopingCartPage} from "../page/index";
+import {LoginPage} from "../page/index";
+import {AddressStepPage} from "../page/index";
+import {ShippingStepPage} from "../page/index";
+import {PaymentStepPage} from "../page/index";
```

Comment:
you can have it all in only one import:
```suggestion
import {
  MenuContentPage,
  ProductslistPage,
  ShopingCartPage,
  LoginPage,
  AdressStepPage,
  ShippingStepPage,
  PaymentStepPage
} from "../page/index";
```